src/website/views/components/forms/checkbox/_examples.pug

```diff
@@ -5,21 +5,25 @@ h3 Base
   .-p--3
     .chi-form__item
       .chi-checkbox
-        input(type="checkbox" class="chi-checkbox__input" id="checkbox-ba1")
-        label(for="checkbox-ba1" class="chi-checkbox__label") Checkbox
+        input(type="checkbox" class="chi-checkbox__input" id="checkbox__base")
+        label(for="checkbox__base" class="chi-checkbox__label") Checkbox
```

Comment:
We are still using HTML Blueprint snippets for our examples, I suggest changing our examples and using web components for all the examples we have.

Remember that you don´t need the `.chi-checkbox` div now.

```suggestion
        chi-checkbox(id="checkbox__base" label="Checkbox")
```